packages/node_modules/@ciscospark/plugin-avatar/src/avatar-url-store.js

```diff
@@ -78,14 +78,13 @@ export default class AvatarUrlStore {
     if (!item.url) {
       return Promise.reject(new Error(`\`item.url\` is required`));
     }
-    if (!item.cacheControl) {
```

Comment:
It looks like we were trying to use item.cacheControl, but didn't bother to parse it. I think in handleItemSuccess you want to look at item.cacheControl, not item.response.cacheControl.